.github/workflows/maven.yaml

```diff
@@ -2,26 +2,34 @@ name: github-workflow
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, 2.1.x ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, 2.1.x ]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       TESTCONTAINERS_REUSE_ENABLE: false
+    outputs:
```

Comment:
in plain english this says: "if I ran on the step related to jdk-17 (which means base branch is `main`), set `jdk_17_branch=true`". I need this because this let's me skip fabric8-istio integration test, cause when we run `2.1.x` - we do not run integration tests. And fabric8-istio, unlike all other integration tests, is a separate step (we need this when we run on `main` as the base branch, basically to start a testcontainer and reuse it later).